services/core/shared/core/delegates.js

```diff
@@ -129,8 +128,7 @@ const loadAllDelegates = async () => {
 };
 
 const loadAllNextForgers = async () => {
-	// TODO: These feature should be handled by the compatibility layer
-	const maxCount = (sdkVersion < 4) ? 101 : 103;
+	const maxCount = await coreApi.getNumberOfForgers();
 	if (sdkVersion <= 4) {
 		rawNextForgers = await requestAll(coreApi.getNextForgers, { limit: maxCount }, maxCount);
 	} else {
```

Comment:
This piece of code can be simplified now. Please keep only the following lines:
```
		const { data } = await coreApi.getForgers({ limit: maxCount, offset: nextForgers.length });
		rawNextForgers = data;
```

Reply:
Done!